presentation/src/main/java/com/bunbeauty/presentation/viewmodel/main/MainAction.kt

```diff
@@ -0,0 +1,14 @@
+package com.bunbeauty.presentation.viewmodel.main
+
+import androidx.navigation.NavController
+import com.bunbeauty.presentation.viewmodel.base.Action
+
+sealed interface MainAction : Action {
```

Comment:
put it in one class

Event , action and dataState

Reply:
fixed ✅